### DISCUSSION TITLE
Is trino   support   direct  querying  data of delta_lake ?

### DISCUSSION THREAD

**Discussion by Rock-520** [trinodb/trino#15032] [Q&A]
When i   used  presto , it hash `$path$` to query  data of delta_lake, but  change to use trino , it has not  `$path$` schema.

```sql
SELECT * FROM sales."$path$"."s3://db-sa-datasets/presto/sales_data" LIMIT 200;
```
https://prestodb.io/docs/current/connector/deltalake.html

**Comment by ebyhr** [comment]
It's not supported in Delta Lake connector of Trino.
